demos/common/cpp/models/include/models/results.h

```diff
@@ -22,9 +22,11 @@
 
 struct MetaData;
 struct ResultBase {
+    ResultBase(int64_t frameId = -1, const std::shared_ptr<MetaData>& metaData = nullptr) :
+        frameId(frameId), metaData(metaData) {}
     virtual ~ResultBase() {}
 
-    int64_t frameId = -1;
+    int64_t frameId;
```

Comment:
Since you are refactoring this, what do you think about `frameId` -> `resultId` (or even `id`)?

Reply:
I'd prefer to keep it as is - this ID is associated with data frames order (first of all, video data frames), so I'd better keep emphasis that this is sequential frame ID, not just general ID of result.